src/app/content/components/__snapshots__/Content.spec.tsx.snap

```diff
@@ -3272,7 +3272,6 @@ li[aria-label="Current Page"] .c60 {
             >
               <input
                 aria-label="Search this book"
-                autoFocus={true}
```

Comment:
but we do want this to become focused if you're on the extra small mobile view where the input is hidden initially and you need to press a button to open it

Reply:
For the record, that does not happen in Production. You press the button to open the search input and you still have to navigate to it.

Reply:
I have added an Effect to the input to grab focus when the mobile toolbar opens.